style.css

```diff
@@ -1180,3 +1214,132 @@ textarea {
   color: #fff;
   background-color: #2230d2;
 }
+
+@media only screen and (min-width: 768px) {
+  .toggle-hidden1 {
+    display: none;
+  }
+
+  .toggle-hidden2 {
+    display: flex;
+  }
+
+  .headline-container1 {
+    background: url(assets/header-bg-desktop.svg);
+    position: relative;
+    width: 100vw;
+    height: 721.46px;
+    left: 34.6px;
+    top: 0;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .CSS-grid__card-container {
+    background: #e5e5e5;
+
+    /* Works */
+
+    /* Auto layout */
+
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    align-items: center;
+    justify-content: center;
+    padding: 124px auto;
+    gap: 142px;
+    position: relative;
+    width: 100vw;
+    height: 2788px;
+    top: 100px;
+  }
+
+  .CSS-grid__card-container .Card-works {
+    /* Card Works */
+
+    box-sizing: border-box;
+
+    /* Auto layout */
+
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    align-self: center;
+    margin-left: 8%;
+    padding: 16px;
+    gap: 2%;
+    width: 80vw;
+    height: 552px;
+
+    /* Neutrals/N0 */
+
+    background: #fff;
+
+    /* Neutrals/N40 */
+
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .snapshoot-container {
+    /* Snapshoot Portfolio */
+
+    width: 50%;
+    height: 80%;
+    border-radius: 8px;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  /* Left Block in Flex Card */
+
+  .left-block {
+    /* Left Block */
+
+    /* Auto layout */
+
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    width: 50%;
+    height: 80%;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .about-myself-container {
+    /* About My Self */
+
+    /* Auto layout */
+
+    margin-top: 114px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 114px 50px;
+    position: relative;
+    width: 100%;
+    height: 600.15px;
+    left: auto;
+    right: auto;
+    top: 0;
+  }
+}
```

Comment:
- [ ] Your desktop version of the about me page looks great, but it is important to make it match the Figma template as closely as possible. Please consider the following suggestions:

- The page should be divided into 2 blocks, with the left block taking up 40% and the right block taking up the remaining 60% of the page's width.
- The "Get my Resume" button needs to be redesigned to match the button design in the template.
- The languages "JavaScript", "HTML", and "CSS" should be added as shown in the Figma template.
- The top right corner should have a rounded border in accordance with the template.
**Your design**

![Screenshot from 2023-03-29 08-22-29](https://user-images.githubusercontent.com/105117832/228552307-20ffd67e-b007-455f-8443-a97eff208b61.png)

**Figma**

![Screenshot from 2023-03-29 08-22-05](https://user-images.githubusercontent.com/105117832/228552348-9d8e6c42-9f98-4e0b-883b-65df2bfbbeac.png)

Reply:
i made the changes but the top right rounded corner and The languages "JavaScript", "HTML", and "CSS"  are yet to be Added.